nucypher/characters/control/specifications/bob.py

```diff
@@ -45,15 +41,15 @@ class Retrieve(BaseSchema):
             type=click.STRING,
             required=False)
     )
-    message_kit = character_fields.UmbralMessageKit(
+    message_kit = character_fields.MessageKit(
         required=True,
         load_only=True,
         click=options.option_message_kit(required=False)
```

Comment:
This should be required

```suggestion
        click=options.option_message_kit(required=True)
```